pulsar-client-admin-api/src/main/java/org/apache/pulsar/common/policies/data/ManagedLedgerInternalStats.java

```diff
@@ -77,6 +77,7 @@ public static class LedgerInfo {
         public boolean offloaded;
         public String metadata;
         public boolean underReplicated;
+        public boolean bookkeeperDeleted;
```

Comment:
We'd better add a compatibility test to cover the newly added field. Refer to: https://github.com/apache/pulsar/pull/21521#issuecomment-1818228947